src/components/cryptolist.module.css

```diff
@@ -0,0 +1,131 @@
+.cryptolist {
+  background: #0b0e1c;
+  color: white;
+}
+
+.unordered {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 47% 47%;
+  gap: 0.4%;
+  padding: 2% 0 2% 3%;
```

Comment:
- [ ] Following the project requirements, you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
  - Typography: font face, size, and weight
  - Layout: composition and space between elements.

<img width="461" alt="image" src="https://user-images.githubusercontent.com/77190203/184117584-a82e912a-461e-4932-8520-99b11a8831eb.png">


- [ ] Your detail page does not match the design guidelines. Please try to follow the provided design as much as you can.
- Your detail page design:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/184117072-95921c7a-020e-4585-828b-c01746280d24.png">


- The provided design:
<img width="126" alt="image" src="https://user-images.githubusercontent.com/77190203/184116758-cf946ddc-b95a-4ea2-99df-9f0cdf76638c.png">